addons/stock/models/product.py

```diff
@@ -453,8 +453,7 @@ def _compute_quantities_dict(self):
         return prod_available
 
     def _search_quantities(self, operator, value):
-        # TDE FIXME: does this work anyway ?
-        domain = []  # TDE ADDED
+        domain = [('qty_available', operator, value)]
```

Comment:
It is not only about qty_available and this thing worked in old api despite the comment